### ISSUE TITLE
[css-backgrounds] Clarify the interaction of background-clip: text with the position property

### ISSUE THREAD

**Issue by yisibl** [w3c/csswg-drafts#9563]
Consider an example(From: https://bugs.chromium.org/p/chromium/issues/detail?id=1339290#c26):

```html
<style>
    div {
        font-size: 36px;
        font-family: sans-serif;
        background-image: linear-gradient( red, blue );
        -webkit-text-fill-color: transparent;
         -webkit-background-clip: text;
        background-clip: text;
    }

    span {
        position: relative;
        border: 2px solid green;
    }
</style>

<div>
    <span>Hello word</span>
</div>
```

The current behavior of Chrome and Safari does not show text, which looks like a bug, and only Firefox is as expected. Considering that [Chrome 120 removed prefixes](https://chromestatus.com/feature/5125388091260928), the specification needs to define a clear behavior as soon as possible.

Also, how do other values interact, not just relative values?

**Comment by Loirooriol** [comment]
Also happens with `opacity: 0.9`, `transform: scale(1)` and `contain: paint` (may need `display: block` to apply). Or an integer `z-index` on a flex item.

**Comment by yisibl** [comment]
@Loirooriol Thanks for providing more cases, I think it basically fails as soon as the [Stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) is created.

**Comment by Loirooriol** [comment]
Yeah, in https://drafts.csswg.org/css-position-4/#paint-a-stacking-context, things that invoke "paint a stacking context" or "paint a stacking container".

**Comment by yisibl** [comment]
I see that WebKit is improving `background-clip`, which is exciting. @smfr Do you have any plans to fix this in WebKit?

cc @fantasai @LeaVerou @SebastianZ

**Comment by yisibl** [comment]
This failed WPT case also appears to be related to the stacking context: https://wpt.fyi/results/css/css-backgrounds/background-clip/clip-text-animated-text.html?label=master&label=experimental&aligned
